client/engine/store/mockstore_test.go

```diff
@@ -145,7 +148,17 @@ func TestConsensusChannelStore(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if err := ms.SetConsensusChannel(&want.ConsensusChannel); err != nil {
+	// Generate a new proposal so we test that the proposal queue is being fetched properly
+	proposedGuarantee := cc.NewGuarantee(big.NewInt(1), types.Destination{2}, left.AsAllocation().Destination, right.AsAllocation().Destination)
+	proposal := cc.NewAdd(2, proposedGuarantee, big.NewInt(1))
+	_, err = leader.Propose(proposal, td.Actors.Alice.PrivateKey)
```

Comment:
You can actually pass in any `LedgerOutcome` to `NewConsensusChannel` (which may not be great in production, but can be used here instead of this hack?)